s3/src/main/scala/akka/stream/alpakka/s3/S3Settings.scala

```diff
@@ -31,27 +31,33 @@ case object DiskBufferType extends BufferType {
 }
 
 object S3Settings {
-  def apply(system: ActorSystem): S3Settings =
-    apply(system.settings.config.getConfig("akka.stream.alpakka.s3"))
+  //def apply(system: ActorSystem): S3Settings = apply(system.settings.config.getConfig("akka.stream.alpakka.s3"))
+
+  def apply(): S3Settings = apply(ConfigFactory.load())
 
   /**
    * Create [[S3Settings]] from a Config subsection.
    */
   def apply(config: Config): S3Settings = new S3Settings(
-    bufferType = config.getString("buffer") match {
+    bufferType = config.getString("akka.stream.alpakka.s3.buffer") match {
       case "memory" => MemoryBufferType
       case "disk" => DiskBufferType
       case _ => throw new IllegalArgumentException("Buffer type must be 'memory' or 'disk'")
     },
-    diskBufferPath = config.getString("disk-buffer-path"),
+    diskBufferPath = config.getString("akka.stream.alpakka.s3.disk-buffer-path"),
     proxy = {
-      if (config.getString("proxy.host") != "") {
-        val scheme = if (config.getBoolean("proxy.secure")) "https" else "http"
-        Some(Proxy(config.getString("proxy.host"), config.getInt("proxy.port"), scheme))
+      if (config.getString("akka.stream.alpakka.s3.proxy.host") != "") {
+        val scheme = if (config.getBoolean("akka.stream.alpakka.s3.proxy.secure")) "https" else "http"
+        Some(
+          Proxy(config.getString("akka.stream.alpakka.s3.proxy.host"),
+                config.getInt("akka.stream.alpakka.s3.proxy.port"),
+                scheme)
+        )
       } else None
     },
-    awsCredentials = AWSCredentials(config.getString("aws.access-key-id"), config.getString("aws.secret-access-key")),
-    s3Region = config.getString("aws.default-region"),
-    pathStyleAccess = config.getBoolean("path-style-access")
+    awsCredentials = AWSCredentials(config.getString("akka.stream.alpakka.s3.aws.access-key-id"),
+                                    config.getString("akka.stream.alpakka.s3.aws.secret-access-key")),
+    s3Region = config.getString("akka.stream.alpakka.s3.aws.default-region"),
+    pathStyleAccess = config.getBoolean("akka.stream.alpakka.s3.path-style-access")
```

Comment:
There's no Java API ones here, in addition to the previous comment in this object, would be great if you could add `.create(ActorSystem)` and `create(Config)` and mark those with `Java API`, thanks!